dll/win32/msgina/lang/de-DE.rc

```diff
@@ -172,17 +172,39 @@ BEGIN
     DEFPUSHBUTTON "OK", IDOK, 115, 179, 50, 14
 END
 
-IDD_SHUTDOWN_FANCY DIALOGEX 0, 0, 275, 135
-STYLE NOT WS_VISIBLE | DS_CENTER | DS_MODALFRAME | DS_SHELLFONT | WS_BORDER | WS_CAPTION | WS_DLGFRAME | WS_POPUP
-CAPTION "ReactOS herunterfahren"
+IDD_SHUTDOWN_FANCY DIALOGEX 0, 0, 208, 122
+STYLE NOT WS_VISIBLE | DS_CENTER | DS_SHELLFONT | WS_BORDER | WS_POPUP
 FONT 8, "MS Shell Dlg"
 BEGIN
-    CONTROL IDI_ROSLOGO, IDC_ROSLOGO, "Static", SS_BITMAP, 0, 0, 275, 54
-    CONTROL IDI_BAR, IDC_BAR, "Static", SS_BITMAP, 0, 44, 275, 4
-    PUSHBUTTON "&Schlafmodus", IDC_BUTTON_SLEEP, 37, 63, 55, 31, BS_MULTILINE | WS_GROUP
-    PUSHBUTTON "&Herunterfahren", IDC_BUTTON_SHUTDOWN, 109, 63, 55, 31, BS_MULTILINE
-    PUSHBUTTON "&Neustart", IDC_BUTTON_REBOOT, 182, 63, 55, 31, BS_MULTILINE
-    PUSHBUTTON "Abbrechen", IDCANCEL, 205, 111, 55, 14, WS_GROUP
+    CONTROL IDI_DLG_BG, IDC_DLG_BG, "STATIC", SS_BITMAP | SS_REALSIZECONTROL, 0, 0, 208, 122
+    CONTROL IDI_ROSLOGO_FLAG, IDC_ROSLOGO_WHITE, "STATIC", SS_BITMAP, 176, 1, 32, 26
+    PUSHBUTTON "&Energiesparmodus", IDC_BUTTON_HIBERNATE, 30093, 30049, 22, 20, BS_OWNERDRAW | WS_DISABLED
```

Comment:
You can disable the buttons by software mechanism.

Reply:
@katahiromz I am aware of it, however I decided to do by resource anyway.